graph/src/data/graphql/effort.rs

```diff
@@ -92,11 +94,8 @@ impl QueryEffort {
     /// for the query
     pub fn current_effort(&self, shape_hash: u64) -> (Option<Duration>, Duration) {
         let inner = self.inner.read().unwrap();
-        let total_effort = inner.total.average().unwrap_or(*ZERO_DURATION);
```

Comment:
An improvement unrelated to this commit: The constructors for `Duration` are `const`, so I believe you can pull some `Duration` constants out of `lazy_static` and into an ordinary constant.

Reply:
Ooh .. nice; I didn't realize that. Added another commit to address that.